examples/sensor/sensor/file_sensor_example.py

```diff
@@ -0,0 +1,49 @@
+# %% [markdown]
+# # File Sensor
+#
+# This example shows how to use the `FileSensor` to detect files appearing in your local or remote filesystem.
+#
+# To begin, import the required libraries.
+
+# %%
+from flytekit import task, workflow
+from flytekit.sensor.file_sensor import FileSensor
+
+# %% [markdown]
+# Create a FileSensor task.
+#
+# The sensor will search for the file at the specified path. If the file exists, it will return a succeed status. Otherwise, the sensor will continue running until the file is added to the directory.
```

Comment:
I recommend changing this to "To use the FileSensor created in the previous step, you must specify the `path` parameter. In the sandbox, you can use the S3 path" and moving the last code block directly below it.